scripts/apps/vocabularies/controllers/constants.ts

```diff
@@ -0,0 +1,6 @@
+export const MEDIA_TYPES = {
+    'media': 'Media gallery',
+    'related_content': 'Related items'
```

Comment:
```
export const MEDIA_TYPES = {
    GALLERY: {
        id: 'GALLERY',
        label: gettext('Media gallery')
    },
    RELATED_CONTENT: {
        id: 'RELATED_CONTENT',
        label: gettext('Related items')
    },
};
```

I think defining it in this way would be better since you could access the id by referring to the object. Like `MEDIA_TYPES.GALLERY.id`. Otherwise, you would have to hardcode the id in the usage. It will not be fun to grep 'media' and expect to find usages.

Reply:
Done